src/script/builder/tir/expr.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#ifndef TVM_SCRIPT_BUILDER_TIR_EXPR_H_
+#define TVM_SCRIPT_BUILDER_TIR_EXPR_H_
+
+#include <tvm/tir/expr.h>
+
+#include "../builder.h"
+
+namespace tvm {
+namespace script {
+namespace builder {
+namespace tir {
+
+using tvm::tir::Add;
+using tvm::tir::And;
+using tvm::tir::Cast;
+using tvm::tir::Div;
+using tvm::tir::EQ;
+using tvm::tir::FloorDiv;
+using tvm::tir::FloorMod;
+using tvm::tir::GE;
+using tvm::tir::GT;
+using tvm::tir::LE;
+using tvm::tir::LT;
+using tvm::tir::Max;
+using tvm::tir::Min;
+using tvm::tir::Mod;
+using tvm::tir::Mul;
+using tvm::tir::NE;
+using tvm::tir::Not;
+using tvm::tir::Or;
+using tvm::tir::Select;
+using tvm::tir::Sub;
```

Comment:
we don't want to import them here directly. In TVM, these IR nodes are constructed via lowercased methods, for example `tvm::tir::Cast` is constructed via `tvm::cast` defined in `op.h`